internal/api/sms_provider/messagebird.go

```diff
@@ -55,6 +56,15 @@ func NewMessagebirdProvider(config conf.MessagebirdProviderConfiguration) (SmsPr
 	}, nil
 }
 
+func (t *MessagebirdProvider) SendMessage(phone string, message string, channel string) error {
+	switch channel {
+	case SMSProvider:
+		return t.SendSms(phone, message)
+	default:
+		return errors.New("channel type is not supported for Vonage")
```

Comment:
I would prefer `fmt.Errorf("channel type %q is not supported for Vonage", channel)` so we know what is getting to it.